cvat-ui/src/cvat-core-wrapper.ts

```diff
@@ -7,6 +7,7 @@ import CVATCore from 'cvat-core/src';
 import _cvat from 'cvat-core/src/api';
 
 import ObjectState from 'cvat-core/src/object-state';
+import { validateAttributeValue } from 'cvat-core/src/object-utils';
```

Comment:
Instead of direct import from cvat-core, please add this function to `cvat.utils` api: cvat/cvat-core/src/api.ts
And update typings here:cvat/cvat-core/src/index.ts

Reply:
E.g. see how it works with `mask2Rle` method